### PR TITLE
Pyportal intake additions

### PR DESCRIPTION
**Pull request by HarmdR**
He Lazlo,
Wat kleine veranderingen ten behoeve van de intake module

Gr,
Harm

### CODE REVIEW

File: intake_dataset.py [hunk 1]
```diff
@@ -52,8 +52,14 @@ def intake_report_export_study_data(ctx, study_id):
                                            "COLL_NAME like '{}%'".format(set_path),
                                            genquery.AS_LIST, ctx)
             for row in result:
+                log.write(ctx, 'export')
```
Comment (lwesterhof) [inline intake_dataset.py]: Leftover debug statements

File: intake_vault.py [hunk 2]
```diff
@@ -138,8 +138,7 @@ def dataset_collection_move_2_vault(ctx, toplevel_collection, dataset_id, vault_
     buffer = {}
     if status == 0:
         # stamp the vault dataset collection with additional metadata
-        date_created = datetime.now()
-        avu.set_on_coll(ctx, vault_path, "dataset_date_created", date_created.strftime('%Y-%m-%dT%H:%M:%S.%f%z'))
+        avu.set_on_coll(ctx, vault_path, "dataset_date_created", str(int(time.time())))
```
Comment (lwesterhof) [inline intake_vault.py:141]: Does this change the way dataset_date_created is stored? How is this handled for existing datasets?

**Review by lwesterhof — CHANGES_REQUESTED** [review]

File: intake_vault.py (hunk 2 above)
Comment (HarmdR) [reply intake_vault.py:141]: Good observation and question!
The timestamp must be stored as a number. This is the way it was originally and I introduced this fuckup myself.
I came across this error when I added the reporting part of this module to the python portal.
I erroroneously added a string formatted (Ymd etc) dataset creation timestamp.

Originally, in youth cohort ruleset, it was:

				# stamp the vault dataset collection with additional metadata
				msiGetIcatTime(*date, "unix");
				msiAddKeyVal(*kv, "dataset_date_created", *date);

As 'Youth' is not working with the new python refactored rules yet , as it was not released yet, no datasets have been added with an erroneous dataset_creation (gelukkig!)  :-)

File: intake_dataset.py (hunk 1 above)
Comment (HarmdR) [reply intake_dataset.py]: well spotted! Totally overlooked them

Corrected 👍

**Review by lwesterhof — APPROVED** [review]

**Pull request merged**